packages/core/test/unit/application.test.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright IBM Corp. 2017. All Rights Reserved. // Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {Application, Server, Component} from '../../index';
+import {Context, Constructor} from '@loopback/context';
+
+describe('Application', () => {
+  describe('start', () => {
+    it('starts all injected servers', async () => {
+      const app = new Application({
+        components: [FakeComponent],
+      });
+
+      await app.start();
+      const server = await app.getServer(FakeServer);
+      expect(server).to.not.be.null();
+      expect(server.running).to.equal(true);
+      await app.stop();
+    });
+    it('does not attempt to start poorly named bindings', async () => {
```

Comment:
Please add an empty line before this test function to server as a visual delimiter.